types/faker-schema/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for faker-schema 1.0
+// Project: https://github.com/knicklabs/faker-schema
+// Definitions by: Yury A Troynov <https://github.com/yutro>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function withProbability<T>(value: (() => T) | T, probability?: number): <S>(schema: S) => (T extends () => void ? S : T) | null;
+
+export class Schema<T> {
```

Comment:
The T has to extend object/record, as per description: `create one or more pseudo-random records from schema`, where a schema created with `A blueprint (..) function that returns an object literal.`
T cannot be anything:
```ts
const bluePrint = () => (41);
const personSchema = new Schema(bluePrint);
const person = personSchema.makeOne();
person.toFixed();
```
(which is legit TS, no errors)

Reply:
added now it extends Object :)